frontend/javascripts/viewer/controller/combinations/volume_handlers.ts

```diff
@@ -132,17 +175,24 @@ export function handlePickCellFromGlobalPosition(
   globalPos: Vector3,
   additionalCoordinates: AdditionalCoordinate[],
 ) {
-  const segmentId = getSegmentIdForPosition(globalPos);
-
-  if (segmentId === 0) {
+  const visibleSegmentationLayer = getVisibleSegmentationLayer(Store.getState());
+  if (visibleSegmentationLayer == null) {
     return;
   }
-  Store.dispatch(setActiveCellAction(segmentId, globalPos, additionalCoordinates));
+  const posInLayerSpace = globalToLayerTransformedPosition(
```

Comment:
`handlePickCellFromGlobalPosition`  no longer needs to be exported. No other file seems to import it